app/controllers/users/settings/user_teams_controller.rb

```diff
@@ -195,7 +195,7 @@ def remove_user_from_team!(user_assignment, new_owner)
           protocol.added_by = new_owner
           protocol.archived_by = new_owner unless protocol.archived_by.nil?
```

Comment:
Why do we update `archived_by/restored_by`, it can be done by other than `added_by`.
And now we also added `last_modified_by`